face_api_dataset/dataset.py

```diff
@@ -11,6 +11,7 @@
 import pandas as pd
 from pkg_resources import parse_version
 from pdb import set_trace
+from scipy.spatial.transform.rotation import Rotation
```

Comment:
Looks like copy is not in the dependencies

Reply:
It may be reasonable to add it as a local include to the corresponding method, so that those who are not using them will not have to install it.